File: Euclid/Messages/Outgoing/SYSTEMBROADCAST.cs
using Euclid.Network.Streams.Util;

namespace Euclid.Messages.Outgoing
{
    class SYSTEMBROADCAST : IMessageComposer
    {
        private string message;

        public SYSTEMBROADCAST(string message)
        {
            this.message = message;
        }

        public override void Write()
        {
            this.Data.Add(new ArgumentEntry(this.message));
        }
    }
}

File: euclid/src/main/java/euclid/messages/outgoing/SYSTEMBROADCAST.java
package euclid.messages.outgoing;

import euclid.messages.IMessageComposer;
import euclid.network.streams.util.ArgumentEntry;

public class SYSTEMBROADCAST extends IMessageComposer {

    private final String message;

    public SYSTEMBROADCAST(String message) {
        this.message = message;
    }

    @Override
    public void write() {
        getData().add(new ArgumentEntry(message));
    }
}
